### PR TITLE
Fix #177379 dallasnews.com

### LINKED ISSUE #177379: www.dallasnews.com
### Issue URL (Ads)
 [https://www.dallasnews.com/news/faith/2024/04/18/dozens-of-north-texas-congregations-have-left-the-united-methodist-church-since-2019/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Fwww.dallasnews.com%2Fnews%2Ffaith%2F2024%2F04%2F18%2Fdozens-of-north-texas-congregations-have-left-the-united-methodist-church-since-2019%2F)
Username: `@arencambre`

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>

### System configuration

Information | Value
--- | ---
AdGuard product: | AdGuard for iOS v4.5.3
License type: | paid
Advanced protection: | enabled
DNS filtering: | disabled
Filters: | <b>Ad Blocking:</b><br/>AdGuard Base, <br/>AdGuard Mobile Ads<br/><br/><b>Privacy:</b><br/>AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/>AdGuard Annoyances

### PR DESCRIPTION
**Pull request by artemiv4nov**
Upd for #177379

### CODE REVIEW

**Review by piquark6046 — COMMENTED** [review]
please rename title of the PR to `Fix #177379 dallasnews.com`.

**Review by BlazDT — CHANGES_REQUESTED** [review]
Ad is self-promo. So please move to section in Annoyances filter.

Also please remove from spyware filter (not seeing anything spyware related).


Please also hide newsletter section:
```
Breaking News
Get the latest breaking news from North Texas and beyond.
```

**Comment by Alex-302** [comment]
@BlazDT 

> Also please remove from spyware filter (not seeing anything spyware related).

Tracking here

**Comment by Alex-302** [comment]
Missed subscription 

<details>
<summary>Details</summary>

image

</details>

**Comment by BlazDT** [comment]
> @BlazDT
> 
> > Also please remove from spyware filter (not seeing anything spyware related).
> 
> Tracking here

Referrer urls shown now, so I stand corrected. Please readd domain to tracking server first party.

**Comment by artemiv4nov** [comment]
> > @BlazDT
> > > Also please remove from spyware filter (not seeing anything spyware related).
> > 
> > 
> > Tracking here
> 
> Referrer urls shown now, so I stand corrected. Please readd domain to tracking server first party.

Done, @BlazDT , please re-check

File: SpywareFilter/sections/mobile.txt [hunk 1]
```diff
@@ -176,6 +176,7 @@ inapps.appsflyersdk.com^
 !
 ! NOTE: Regular rules
 !
+||newsstat.dallasnews.com^
```
Comment (BlazDT) [inline SpywareFilter/sections/mobile.txt:179]: Request happens on desktop too, so please move to tracking_servers_firstparty

**Review by BlazDT — CHANGES_REQUESTED** [review]

File: SpywareFilter/sections/mobile.txt (hunk 1 above)
Comment (artemiv4nov) [reply SpywareFilter/sections/mobile.txt]: Done

**Review by BlazDT — APPROVED** [review]

**Review by AdamWr — APPROVED** [review]

**Pull request merged**